src/routes/JobOpportunity/JobOpportunity.tsx

```diff
@@ -1,9 +1,217 @@
+            {t('job-opportunity.most-common-job-tasks.title')}
+          </h2>
+          <p className="text-body-md font-arial my-5">{t('job-opportunity.most-common-job-tasks.description')}</p>
+          <ol className="list-decimal ml-7 text-body-lg font-medium text-black leading-7">
+            {ammatit.map((ammatti) => (
+              <li
+                className="text-capitalize text-body"
+                title={`${ammatti.koodi} ${getLocalizedText(ammatti.kuvaus)} (${ammatti.osuus.toFixed(1)}%, N = ${clusterSize})`}
+                key={ammatti.uri}
+              >
+                {getLocalizedText(ammatti.nimi)}
+              </li>
+            ))}
+          </ol>
+        </div>
+        <div>
+          <h2 id={t('job-opportunity.key-figures.title')} className="text-heading-2 scroll-mt-[96px]">
+            {t('job-opportunity.key-figures.title')}
+          </h2>
+          <p className="text-body-md font-arial mb-6 mt-4">{t('job-opportunity.key-figures.description')}</p>
+          <div className="bg-todo h-[380px]" />
+        </div>
+        <div>
+          <h2 id={t('job-opportunity.labour-market-picture.title')} className="text-heading-2 scroll-mt-[96px]">
+            {t('job-opportunity.labour-market-picture.title')}
+          </h2>
+          <p className="text-body-md font-arial mb-6 mt-4">{t('job-opportunity.labour-market-picture.description')}</p>
+          <div className="bg-todo h-[380px]" />
+        </div>
+        <div>
+          <h2 id={t('job-opportunity.salary-trends.title')} className="text-heading-2 scroll-mt-[96px]">
+            {t('job-opportunity.salary-trends.title')}
+          </h2>
+          <p className="text-body-md font-arial mb-6 mt-4">{t('job-opportunity.salary-trends.description')}</p>
+          <div className="bg-todo h-[380px]" />
+        </div>
+        <div>
+          <h2 id={t('job-opportunity.competences.title')} className="text-heading-2 scroll-mt-[96px]">
+            {t('competences')}
+          </h2>
+          <CompareCompetencesTable
+            opportunityName={tyomahdollisuus?.otsikko}
+            rows={competencesTableData}
+            className="mt-4"
+          />
+        </div>
+        <div>
+          <h2 id={t('job-opportunity.employment-trends.title')} className="text-heading-2 scroll-mt-[96px]">
+            {t('job-opportunity.employment-trends.title')}
+          </h2>
+          <p className="text-body-md font-arial mb-6 mt-4">{t('job-opportunity.employment-trends.description')}</p>
+          <div className="bg-todo h-[380px]" />
+        </div>
+        <div>
+          <h2 id={t('job-opportunity.related-jobs.title')} className="text-heading-2 scroll-mt-[96px]">
+            {t('job-opportunity.related-jobs.title')}
+          </h2>
+          <p className="text-body-md font-arial mb-6 mt-4">{t('job-opportunity.related-jobs.description')}</p>
+          <div className="bg-todo h-[380px] mb-8" />
+        </div>
```

Comment:
And while this is just a todo block and such, we still could improve readability for the time being by using component:

```tsx
const WipBlock = ({ label }: { label: string }) => {
  return (
    <>
      <p className="text-body-md font-arial mb-6 mt-4">{label}</p>
      <div className="bg-todo h-[380px]" />
    </>
  );
};
```

Usage (together with that H2Scroll ):

```tsx
<div className="mb-8">
  <H2Scroll title={t('job-opportunity.related-jobs.title')} />
  <WipBlock label={t('job-opportunity.related-jobs.description')} />
</div>
```

Reply:
A solid idea, but I feel it's not worth the effort to start wrapping todo-stuff in todo-components for the sake of readability. Maybe if there were more of these todo blocks.